src/unity/python/turicreate/visualization/__init__.py

```diff
@@ -3,6 +3,121 @@
 #
 # Use of this source code is governed by a BSD-3-clause license that can
 # be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+#
+# Turi Create Visualization allows you to create Plot objects and 
+# manipulate them using various functionalities, including
+# saving them (.save) and viewing them (.show). Moreover, Turi Create 
+# Visualization also allows the user to call `show` on data sources like 
```

Comment:
Be consistent - are we calling the audience "you" or "the user"? I think the former.